napari/layers/points/_points_utils.py

```diff
@@ -45,17 +86,85 @@ def points_to_squares(points, sizes):
     """
     rect = np.concatenate(
         [
-            points + np.sqrt(2) / 2 * np.array([sizes, sizes]).T,
-            points + np.sqrt(2) / 2 * np.array([sizes, -sizes]).T,
-            points + np.sqrt(2) / 2 * np.array([-sizes, sizes]).T,
-            points + np.sqrt(2) / 2 * np.array([-sizes, -sizes]).T,
+            points + 0.5 * np.array([sizes, sizes]).T,
+            points + 0.5 * np.array([sizes, -sizes]).T,
+            points + 0.5 * np.array([-sizes, sizes]).T,
+            points + 0.5 * np.array([-sizes, -sizes]).T,
```

Comment:
I don't see where `sizes` is defined here, am I missing something?

Reply:
Ah yeah, it is a little confusing how github displayed the diff. This is from the function `points_to_squares()`, not `create_box()`. It just looks like it is `create_box()` because the `def points_to_squares()` part is folded. You can either view the whole file or click the unfold button on the left side to see the whole function.

Reply:
my bad!